packages/@webex/plugin-meetings/src/breakouts/breakout.ts

```diff
@@ -111,6 +111,31 @@ const Breakout = WebexPlugin.extend({
   },
 
   /**
+   * assign participants to breakout session
+   * @param {Array} sessions
+   * @param {object} groupPayload
+   * @returns {void}
+   */
+  assign(sessions: any[], groupPayload: any = {}) {
```

Comment:
issue: Given that this is the public API, since you have used it in the sample app, there is too much state passed in or hard coded.

The breakout.assign method should only take a member (or members). The other attributes necessary to control the groups should already be stored on the breakout object and should not be changed during assigning.

i.e. allowBackToMain, allowToJoinLater etc, all of these should not be hardcoded. 

Additionally, the groupPayload should not be passed in. Only members should be passed in. The breakout object is a 1-1 mapping of a session. It has a sessionId already. You shouldn't need to pass in sessions either.